### ISSUE TITLE
🐞 [Bug Report] - querySelectorAll errors in Chrome console

### ISSUE THREAD

**Issue by glasnt** [GoogleCloudPlatform/avocano#161]
**Describe the bug**

```
caught TypeError: Cannot read properties of null (reading 'querySelectorAll')
lit-element-router.js:137
```

**To Reproduce**
1. Go to any deployment
2. Open console
3. Get error.

**Expected behavior**

No error

**Screenshots**

<img width="893" alt="Screenshot 2023-04-12 at 4 35 46 pm" src="https://user-images.githubusercontent.com/813732/231371566-69147459-5223-432e-b528-166df16f253f.png">

**Additional context**

Not to be confused with #159, which is a different error.

**Comment by glasnt** [comment]
This in an issue in lit-element-router.js

Identifiable when running locally:

<img width="898" alt="Screenshot 2023-04-12 at 4 35 31 pm" src="https://user-images.githubusercontent.com/813732/231371669-c275a362-98f8-48e3-91dc-3b623feb624a.png">

This package hasn't been updated in a while, and there's an unmerged PR that when manually applied fixes the bug: https://github.com/hamedasemi/lit-element-router/pull/47

**Comment by glasnt** [comment]
@pattishin would appreciate your insight here around patching, finding an alternative package, etc.

**Comment by bivens-dev** [comment]
Would you consider using the router package that the Lit team themselves are developing. 

I’ve used it myself and found it to be a really nice and easy to work with API that supports some nice features such as the native URLPattern spec for specifying routes and other incremental loading of components as needed by a given route.

I know they are also looking for feedback as well and thought it might be a good match. https://github.com/lit/lit/tree/main/packages/labs/router

**Comment by pattishin** [comment]
This was due to the `shadowRoot` not being loaded in time. This will be resolved by glasnt's https://github.com/GoogleCloudPlatform/avocano/pull/290.

**Comment by glasnt** [comment]
Resolved in #290

**Comment by glasnt** [comment]
This error no longer occurs in either buynow or cart deployments (manually tested)
